<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>756</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="6" column="0" colspan="5">
     <widget class="QPlainTextEdit" name="plainTextEdit">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>150</height>
       </size>
      </property>
      <property name="sizeIncrement">
       <size>
        <width>1</width>
        <height>1</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Courier New</family>
        <pointsize>8</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="cursor" stdset="0">
       <cursorShape>ArrowCursor</cursorShape>
      </property>
      <property name="toolTip">
       <string>Useful information</string>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="lineWidth">
       <number>1</number>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="textInteractionFlags">
       <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QProgressBar" name="progressBar">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>0</height>
       </size>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
    </item>
    <item row="7" column="2">
     <widget class="QPushButton" name="pushButton_4">
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="toolTip">
       <string>Tell wit to create a image using the selected parameters</string>
      </property>
      <property name="text">
       <string>Do It</string>
      </property>
     </widget>
    </item>
    <item row="7" column="3">
     <widget class="QPushButton" name="pushButton_2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="toolTip">
       <string>Clear the text</string>
      </property>
      <property name="text">
       <string>Clear</string>
      </property>
     </widget>
    </item>
    <item row="7" column="4">
     <widget class="QPushButton" name="pushButton_3">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="toolTip">
       <string>Copy the text in the output box</string>
      </property>
      <property name="text">
       <string>Copy</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="0" colspan="5">
     <widget class="QTabWidget" name="tabWidget">
      <property name="minimumSize">
       <size>
        <width>735</width>
        <height>464</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="mouseTracking">
       <bool>false</bool>
      </property>
      <property name="currentIndex">
       <number>2</number>
      </property>
      <property name="movable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Compose Image</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <layout class="QFormLayout" name="formLayout_2">
          <property name="fieldGrowthPolicy">
           <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
          </property>
          <item row="1" column="0">
           <widget class="QToolButton" name="toolButton">
            <property name="toolTip">
             <string>Enter the path to the files to be used to make the disc image</string>
            </property>
            <property name="text">
             <string>Browse...</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="lineEdit">
            <property name="minimumSize">
             <size>
              <width>250</width>
              <height>0</height>
             </size>
            </property>
            <property name="toolTip">
             <string>Enter the path to the files to be used to make the disc image</string>
            </property>
            <property name="text">
             <string>&lt;input&gt;</string>
            </property>
            <property name="dragEnabled">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QToolButton" name="toolButton_2">
            <property name="toolTip">
             <string>Enter the output file path</string>
            </property>
            <property name="text">
             <string>Browse...</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="lineEdit_2">
            <property name="minimumSize">
             <size>
              <width>250</width>
              <height>0</height>
             </size>
            </property>
            <property name="toolTip">
             <string>Enter the output file path</string>
            </property>
            <property name="text">
             <string>&lt;output&gt;</string>
            </property>
            <property name="dragEnabled">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Input:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Output:</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="0" column="2" rowspan="2">
         <widget class="Line" name="line_3">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
        <item row="0" column="3" rowspan="2">
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QComboBox" name="comboBox">
            <property name="toolTip">
             <string>Set the region used for the output image.  This is independent of the TID.  If auto is selected, wit will try to guess the region based on the TID.</string>
            </property>
            <property name="currentIndex">
             <number>2</number>
            </property>
            <item>
             <property name="text">
              <string>Region &lt;auto&gt;</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>JAPAN</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>USA</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>EUROPE</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>KOREA</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <layout class="QFormLayout" name="formLayout_6">
            <item row="0" column="0">
             <widget class="QLabel" name="label">
              <property name="toolTip">
               <string>What IOS should be used in the TMD</string>
              </property>
              <property name="text">
               <string>Game IOS</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QSpinBox" name="spinBox">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="toolTip">
               <string>What IOS should be used in the TMD</string>
              </property>
              <property name="minimum">
               <number>3</number>
              </property>
              <property name="maximum">
               <number>254</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item row="2" column="0" colspan="4">
         <widget class="Line" name="line_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="3" column="2" colspan="2">
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="1">
           <widget class="QCheckBox" name="checkBox_2">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="toolTip">
             <string>Apply title &amp; TID changes to the disc header</string>
            </property>
            <property name="text">
             <string>Disc Hdr</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QCheckBox" name="checkBox_3">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="toolTip">
             <string>Apply the title &amp; TID changes to the partition header</string>
            </property>
            <property name="text">
             <string>Partition Hdr</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <property name="tristate">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QCheckBox" name="checkBox_4">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="toolTip">
             <string>Apply the new TID to the TMD &amp; ticket</string>
            </property>
            <property name="text">
             <string>Tmd / Ticket</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="3" column="0">
         <layout class="QGridLayout" name="gridLayout_6">
          <item row="0" column="1">
           <widget class="QCheckBox" name="checkBox_6">
            <property name="toolTip">
             <string>Do you want to change the title of the game?</string>
            </property>
            <property name="text">
             <string>Update Title</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QCheckBox" name="checkBox_7">
            <property name="toolTip">
             <string>Do you want to change the TID of the game?</string>
            </property>
            <property name="text">
             <string>Update ID</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLineEdit" name="lineEdit_4">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="minimumSize">
             <size>
              <width>220</width>
              <height>0</height>
             </size>
            </property>
            <property name="toolTip">
             <string>Enter a new game name</string>
            </property>
            <property name="text">
             <string>&lt;Game Name&gt;</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLineEdit" name="lineEdit_3">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="minimumSize">
             <size>
              <width>220</width>
              <height>0</height>
             </size>
            </property>
            <property name="toolTip">
             <string>Enter a new TID ( 6 characters )</string>
            </property>
            <property name="text">
             <string>&lt;ID&gt;</string>
            </property>
            <property name="maxLength">
             <number>6</number>
            </property>
            <property name="dragEnabled">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="Edit_tab">
       <attribute name="title">
        <string>Edit Image</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="label_edit_id">
              <property name="minimumSize">
               <size>
                <width>200</width>
                <height>0</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_edit_region">
              <property name="minimumSize">
               <size>
                <width>100</width>
                <height>0</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QLabel" name="label_edit_name">
            <property name="minimumSize">
             <size>
              <width>475</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="0" column="2">
         <widget class="QPushButton" name="edit_img_pushButton">
          <property name="minimumSize">
           <size>
            <width>110</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>Load Image</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="3">
         <widget class="QTreeWidget" name="treeWidget">
          <property name="font">
           <font>
            <pointsize>8</pointsize>
           </font>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Sunken</enum>
          </property>
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::ExtendedSelection</enum>
          </property>
          <property name="verticalScrollMode">
           <enum>QAbstractItemView::ScrollPerItem</enum>
          </property>
          <property name="indentation">
           <number>20</number>
          </property>
          <property name="sortingEnabled">
           <bool>true</bool>
          </property>
          <property name="animated">
           <bool>true</bool>
          </property>
          <column>
           <property name="text">
            <string>Name</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Offset</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Size</string>
           </property>
          </column>
         </widget>
        </item>
        <item row="0" column="1">
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="settings_tab">
       <attribute name="title">
        <string>Settings</string>
       </attribute>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>100</y>
          <width>391</width>
          <height>161</height>
         </rect>
        </property>
        <layout class="QFormLayout" name="formLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Text output:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="verbose_combobox">
           <property name="currentIndex">
            <number>5</number>
           </property>
           <property name="frame">
            <bool>true</bool>
           </property>
           <item>
            <property name="text">
             <string>Normal</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Quiet</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Verbose</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Verbose+</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Verbose++</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Verbose+++</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Logging :</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QComboBox" name="logging_combobox">
           <property name="currentIndex">
            <number>0</number>
           </property>
           <item>
            <property name="text">
             <string>Off</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Disc</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Disc + Part</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="def_ios_lbl">
           <property name="toolTip">
            <string>Default IOS to use for booting: If this is less than 3, the IOS in the TMD will be used</string>
           </property>
           <property name="text">
            <string>Default IOS:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QSpinBox" name="default_ios_spinbox">
           <property name="toolTip">
            <string>Default IOS to use for booting: If this is less than 3, the IOS in the TMD will be used</string>
           </property>
           <property name="maximum">
            <number>255</number>
           </property>
           <property name="value">
            <number>36</number>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QPushButton" name="pushButton_settings_searchPath">
           <property name="minimumSize">
            <size>
             <width>75</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Search Path...</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="lineEdit_default_path">
           <property name="minimumSize">
            <size>
             <width>224</width>
             <height>0</height>
            </size>
           </property>
           <property name="toolTip">
            <string notr="true"/>
           </property>
           <property name="inputMask">
            <string notr="true"/>
           </property>
           <property name="text">
            <string notr="true">/media</string>
           </property>
           <property name="placeholderText">
            <string notr="true"/>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Start at:</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QComboBox" name="startupTab_combobox">
           <property name="minimumSize">
            <size>
             <width>107</width>
             <height>0</height>
            </size>
           </property>
           <item>
            <property name="text">
             <string>Compose</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Edit</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Settings</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>391</width>
          <height>80</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QCheckBox" name="checkBox">
           <property name="text">
            <string>Test Mode</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="overwrite_checkbox">
           <property name="toolTip">
            <string>Tell wit to overwrite existing files</string>
           </property>
           <property name="text">
            <string>Overwrite Existing</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="checkBox_hiddenFiles">
           <property name="toolTip">
            <string>Skip &quot;.svn&quot; files in tree view to speed up parsing</string>
           </property>
           <property name="text">
            <string>Ignore svn Files in Filetree</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QPushButton" name="save_pushButton">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>400</y>
          <width>115</width>
          <height>28</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>115</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Save Settings</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
